test/models/trip_test.dart

```diff
@@ -0,0 +1,203 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:motis_mitfahr_app/util/search/position.dart';
+import 'package:motis_mitfahr_app/util/trip/trip.dart';
+
+import '../util/factories/mock_trip_factory.dart';
+import '../util/mock_server.dart';
+import '../util/mock_server.mocks.dart';
+
+void main() {
+  final MockUrlProcessor tripProcessor = MockUrlProcessor();
```

Comment:
can you rebase? then this needs to be a MockRequestProcessor if i am correct

Reply:
as written above the rebase is broken 😢  Maybe I'll have to open a new branch and copy the changes by hand.

Reply:
ah sorry my mistake. You could do that i guess or is there any other way?

Reply:
maybe that's the best way idk if there is another one